lib/diagnostic.rb

```diff
@@ -62,8 +74,15 @@ def initialize
 
 ##
 # your answers here
-Answer.housecat_noise = nil
-#
+Answer.housecat_noise = "I am a HouseCat, and I go meow"
+
+# This would be the output because the class HouseCat is inheritted from the
+# class Cat,which is inheritted from the class Animal, which has access to the
+# animal methods. The instance variable sound for HouseCat is equal to a string
+# that says meow. HouseCat inherits the say_hello method from the Animal class
+# and is able to evaluate that method using it's own declared instance variable
+# for sound.
```

Comment:
All correct, particularly the following
> HouseCat inherits the say_hello method from the Animal class and is able to evaluate that method using it's own declared instance variable for sound.

Since it's HouseCat calling the method, that's also why `self` refers to HouseCat, and not Animal.